doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -291,7 +289,10 @@ Update Grub for the Ubuntu Service VM
 Enable Network Sharing for the User VM
 ======================================
 
-In the Ubuntu Service VM, enable network sharing for the User VM:
+In the Ubuntu Service VM, if you need to access the internet, firstly, you need
+to connect Ethernet cable with Ethernet USB Adapter because it doesn't recognize
+the network interface card with the kernel; And then enable network sharing for
```

Comment:
Do we know why it is not recognized? What is the Ethernet adapter in question that's not recognized?

Reply:
It may due to current acrn-kernel not support TGL NUC network interface card, update kernel maybe fix this issue. But for v2.5 release, we just try Ethernet USB Adapter works.

Reply:
I believe this is because we need to turn `CONFIG_IG` on in our kernel config. Can you apply this patch to the `acrn-kernel` repo and try again with the on-board Ethernet controller?
```
$ git diff
diff --git a/kernel_config_uefi_sos b/kernel_config_uefi_sos
index a8095aa3186e..0a5fc2c68f43 100644
--- a/kernel_config_uefi_sos
+++ b/kernel_config_uefi_sos
@@ -351,6 +351,7 @@ CONFIG_E1000=y
 CONFIG_E1000E=m
 CONFIG_IGB=y
 CONFIG_IXGBEVF=y
+CONFIG_IGC=m
 # CONFIG_NET_VENDOR_MARVELL is not set
 # CONFIG_NET_VENDOR_MELLANOX is not set
 # CONFIG_NET_VENDOR_MICREL is not set

```

Reply:
It works, and the document has been updated.